cli/cdk/cdk_grpc.pb.go

```diff
@@ -0,0 +1,147 @@
+// Code generated by protoc-gen-go-grpc. DO NOT EDIT.
```

Comment:
Should we move autogen code to a subfolder? 

cli/cdk/build/cdk_grpc.pb.go ?
cli/cdk/generated/cdk_grpc.pb.go ?

Reply:
any reason why?

Reply:
It's clear from a glance at a project what is autogen and what is not. Take this PR for example, without having to click into the file and read the comment I would know to ignore this file. This is fine for 1 file in a PR, but what about > 10 autogen files living among code that we should be reviewing.

Reply:
moved them 👍🏽